olclient/schemata/edition.schema.json

```diff
@@ -15,6 +15,7 @@
     "key": { "$ref": "shared_definitions.json#/edition_key" },
     "title":    { "type": "string" },
     "subtitle": { "type": "string" },
+    "full_title": { "type": "string" },
```

Comment:
Is this a useful field? It exists on some records, but is not listed at https://openlibrary.org/type/edition

In OL code `full_title` appears to be simply formed from `title` + `subtitle`:

https://github.com/internetarchive/openlibrary/blob/071ca3942302a054355257243930a7a6ddb38d57/openlibrary/core/vendors.py#L406

If it doesn't have another meaning or purpose, it doesn't seem very useful.
If it does have another meaning or purpose, it's not named very well.